service.subtitles.subdivx/service.py

```diff
@@ -584,10 +591,10 @@ def main():
         # Send end of directory to XBMC
         xbmcplugin.endOfDirectory(int(sys.argv[1]))
 
-        _cleanup_tempdir(workdir)
+        time.sleep(2)
```

Comment:
please do not use the time module. Use either `xbmc.sleep()` (small time steps) or preferably `xbmc.Monitor().waitforabort()`

Reply:
Thanks for the review!

I think I've adressed the issue. Followed advice from https://kodi.wiki/view/Service_add-ons

Reply:
yeap, perfect. Thank you